### DISCUSSION TITLE
Update Store API Call for adding users

### DISCUSSION THREAD

**Discussion by niko-underwood** [btcpayserver/btcpayserver#3419] [General]
I'm looking to update the API call for "Update Store" to have the ability to add an existing BTCPayServer user to an existing store. Currently there is only an API call to create a new user. Here is a link to the update store API call for reference: https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Stores_UpdateStore

Use case:

A 3rd party host has a frontend form asking the user to for their name, email, password, and new store they would like to create. Ideally this form data can all be passed via API to create the user, store, and add the user to that store.

Thanks!

**Comment by Kukks** [comment]
https://github.com/btcpayserver/btcpayserver/issues/3426
